x-pack/legacy/plugins/infra/public/components/source_configuration/log_columns_configuration_panel.tsx

```diff
@@ -70,30 +95,30 @@ export const LogColumnsConfigurationPanel: React.FunctionComponent<
 
 interface LogColumnConfigurationPanelProps {
   logColumnConfigurationProps: LogColumnConfigurationProps;
+  dragHandleProps: any;
```

Comment:
EUI doesn't export a type for `dragHandleProps`; I figured using `any` is the best option until that changes so as not to break type checking if EUI changes its `dragHandleProps`